### PR TITLE
Enhance layout Table Operations

### PR DESCRIPTION
**Pull request by karlvaniseghem**
Remove boxContainer and set fixed width on operation tables
Removed extra whitespace around Go buttons
see #13145

### CODE REVIEW

File: tbl_operations.php [hunk 1]
```diff
@@ -299,7 +299,7 @@
 /**
  * Displays the page
  */
-$response->addHTML('<div id="boxContainer" data-box-width="300">');
+$response->addHTML('<div style="min-width: 450px; max-width: 450px; margin: 20px;">');
```
Comment (mauriciofauth) [inline tbl_operations.php]: I don't think it's a good idea to include inline style.

**Comment by nijel** [comment]
Honestly I'm not sure if it's better to leave so much of empty space on the page. Though I agree that current looks is not good. Some feedback from @phpmyadmin/developers is welcome as well.

Current look:

screenshot-2017-6-6 localhost localhost bar2 cars phpmyadmin 4 8 0-dev-current

With this PR:

screenshot-2017-6-6 localhost localhost bar2 cars phpmyadmin 4 8 0-dev

**Comment by nijel** [comment]
PS: The additional spaces around go buttons probably could be removed generally, not just in this specific case...

**Comment by bakgat** [comment]
How about collapsible panels or tabs instead for viewing each fieldset at once ?

**Comment by nijel** [comment]
I've changed default rendering of the fieldset buttons to be more compact in ae67d1c2a7, so at least this part is now better:

screenshot-2017-6-6 localhost localhost bar2 cars phpmyadmin 4 8 0-dev-master

As for layout, I'm not really sure what to choose here. How about using second level navigation to split the page?

**Comment by nijel** [comment]
I've simplified the code in edad6fc2d9a336ed73aab52629479a2a8dfaed9a, any further improvements to the layout should be done on top of that.

**Comment by ibennetch** [comment]
One thing that keeps bothering me is how the first and third items are
nearly the same - copy and move are similar functions, I wonder if we
should try to merge these two fieldsets.

**Pull request closed without merging**